### PR TITLE
first commit of About page

### PR DESCRIPTION
**Pull request by whitney-mitchell**
Change it!

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        <li><a href="http://www.Wagtastic.com/Products" title="Products">Products</a></li>
+        <li><a href="http://www.Wagtastic.com/About_Us" title="About_Us">About Us</a></li>
+        <li><a href="http://www.Wagtastic.com/Contact" title="Contact">Contact</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <div class= "about-content">
+    <img src="https://pixabay.com/static/uploads/photo/2012/12/24/08/40/dog-72333_960_720.jpg" alt="pup!">
+    <h2>Who We Are</h2>
```
Comment (glenngineer1) [inline about.html]: Do we need a semi-colon after Are? Are:

File: about.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      <ul>
+        <li><a href="http://www.Wagtastic.com/Products" title="Products">Products</a></li>
+        <li><a href="http://www.Wagtastic.com/About_Us" title="About_Us">About Us</a></li>
+        <li><a href="http://www.Wagtastic.com/Contact" title="Contact">Contact</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <div class= "about-content">
+    <img src="https://pixabay.com/static/uploads/photo/2012/12/24/08/40/dog-72333_960_720.jpg" alt="pup!">
```
Comment (aoswalt) [inline about.html:26]: p tags are indented an extra time. Second closing p tag has wandered away from the line.

**Comment by glenngineer1** [comment]
Other than the one comment everything else looked good.

**Comment by aoswalt** [comment]
:+1:

**Pull request merged**